tools/mason/MasonBuild.chpl

```diff
@@ -107,72 +107,64 @@ private proc checkChplVersion(lockFile : borrowed Toml) throws {
 proc buildProgram(release: bool, show: bool, force: bool, skipUpdate: bool,
                   ref cmdLineCompopts: list(string), tomlName="Mason.toml",
                   lockName="Mason.lock") throws {
+  const cwd = here.cwd();
+  const projectHome = getProjectHome(cwd, tomlName);
+  const toParse = open(projectHome + "/" + lockName, ioMode.r);
```

Comment:
This string concat happens at least twice. Can you use `Path.joinPath` and store it in a variable instead?